dist/colorpickr.js

```diff
@@ -428,7 +426,7 @@ var ColorPickr = function (_React$Component) {
               ),
               _react2.default.createElement(
                 'div',
-                theme(10, 'inputModeContainer', '' + (colorAttribute === 'b' ? 'active' : '')),
+                theme(11, 'inputModeContainer', '' + (colorAttribute === 'b' ? 'active' : '')),
```

Comment:
You totally don't need to do it in this PR but I'm sure it was annoying trying to keep track of the numbers and adding them. It's possible this [doesn't strictly need to be an integer](https://github.com/markdalgleish/react-themeable/blob/master/src/index.js#L16) and it might be more maintainable if we gave these unique string values instead. i.e

``` js
// Before
theme(11, 'inputModeContainer');

// After
theme('inputModeContainerB', 'inputModeContainer');
```